third-party-solutions/openshift/tf/bootstrap.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_storage_bucket" "bootstrap-ignition" {
+  project       = var.service_project.project_id
+  name          = local.infra_id
+  location      = var.region
+  force_destroy = true
+}
+
+resource "google_storage_bucket_object" "bootstrap-ignition" {
+  for_each = toset(local.bootstrapping ? [""] : [])
+  bucket   = google_storage_bucket.bootstrap-ignition.name
+  name     = "bootstrap.ign"
+  source   = "${local.fs_paths.config_dir}/bootstrap.ign"
+}
+
+data "google_storage_object_signed_url" "bootstrap-ignition" {
+  for_each    = toset(local.bootstrapping ? [""] : [])
+  bucket      = google_storage_bucket.bootstrap-ignition.name
+  path        = google_storage_bucket_object.bootstrap-ignition[""].name
+  credentials = file(local.fs_paths.credentials)
+}
+
+resource "google_compute_instance" "bootstrap" {
+  count        = local.bootstrapping ? 1 : 0
```

Comment:
In the previous two resources you used for_each for the conditional. Pick one for consistency

Reply:
switched all to `count` for better legibility when referencing